helm/templates/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+{{ if $.Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}
+  {{- if $.Values.ingress.annotations }}
+  annotations: {{- $.Values.ingress.annotations | toYaml | nindent 4 }}
+  {{- end }}
+  labels:
+    turboEnigma: {{ .Release.Name }}
```

Comment:
Is this a good moment to change those labels to something more structured, like this https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ ?

So these labels could become `app.kubernetes.io/name: turbo-enigma` or event `app.kubernetes.io/name: {{ $.Chart.Name }}`

Reply:
PS: Changing the labels will force the recreation of the Deployment as some of these fields are immutable.

Reply:
I think it can go in another merge request.

It's good to have an issue for it to keep track of the why.

Reply:
Issue created here https://github.com/codelicia/turbo-enigma/issues/32